src/children.js

```diff
@@ -1,6 +1,2 @@
 import "./treecle-setup.js";
-import { children, childPaths, replace } from "../lib/treecle/src/index.js";
-
-export { children as of, childPaths as paths, replace };
-export default children;
-
+export { children, childPaths, replace as replaceChild } from "../lib/treecle/src/index.js";
```

Comment:
I think it's fine to leave `replace` as-is, what else could it mean? Not to mention that `replaceChild` might be mistaken for a shallow replace (remember when I was confused about this?).